packages/ui/src/theme/overwrites/components/input.ts

```diff
@@ -0,0 +1,32 @@
+export const Input = {
+  sizes: {
+    sm: {
+      field: {
+        padding: '0.375rem 0.75rem',
```

Comment:
It can be better to use pixels for anything non-related to fonts. This way it would be easier for us to quickly check if values match mockups.